docs/index.md

```diff
@@ -33,17 +33,14 @@ better alternative.
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
```

Comment:
This shouldn't be part of this PR. You probably just ran "make docs" and it did this automatically. The CI step that runs make docs keeps the domain in the required section but if you have a specific version of terraform and go locally it will move it to the optional block. I plan to fix this in the upcoming days, but for now let's discard this change please 🙏🏻